### ISSUE TITLE
HN out of range"

### ISSUE THREAD

**Issue by mapomatic** [WazeUSA/WME-Place-Harmonizer#125]

**Comment by t0cableguy** [comment]
Does that really need to be changed for a place that is locked? Its a courtesy check and its not always right.

**Comment by mapomatic** [comment]
Yeah, I think yellow, and change to green if locked makes sense.  The banner highlight should match the map highlight either way.
